src/VisualStudio/Core/Def/Implementation/ProjectSystem/MiscellaneousFilesWorkspace.HostProject.cs

```diff
@@ -60,6 +60,8 @@ public ProjectInfo CreateProjectInfoForCurrentState()
                     metadataReferences: _metadataReferences,
                     isSubmission: false,
                     hostObjectType: null);
+
+                return info.WithIsComplete(isComplete: false);
```

Comment:
You didn't mean true? MiscellaneousFiles to me is the canonical example of this should be set to true, since we know semantics are going to be busted.